root/usr/share/nethesis/NethServer/Module/ProxyPass/Modify.php

```diff
@@ -54,7 +54,7 @@ public function validate(\Nethgui\Controller\ValidationReportInterface $report)
             $report->addValidationErrorMessage($this, 'Name', 'Name_Already_Exists');
         }
 
-        if ( ! filter_var($this->parameters['Target'], FILTER_VALIDATE_URL) || strpos($this->parameters['Target'], 'http') !== 0 ) {
+        if ( ! filter_var($this->parameters['Target'], FILTER_VALIDATE_URL) || (strpos($this->parameters['Target'], 'http') !== 0 ) && ( strpos($this->parameters['Target'], 'ws') !== 0)) {
```

Comment:
I think it's negative logic: if not FILTER_VALIDATE_URL OR (string doesn't start with http AND string doesn't start with ws ) -> validate error
Am I wrong?